src/main/java/htsjdk/tribble/index/AbstractIndex.java

```diff
@@ -67,7 +70,7 @@
     private final static long NO_TS = -1L;
 
     protected int version;                    // Our version value
-    protected File indexedFile = null;         // The file we've created this index for
+    protected Path indexedPath = null;         // The file we've created this index for
```

Comment:
Oh, I haven't realize that it is breaking compatibility. If we do not want to break compatibility, I can kept the field with the deprecated annotation and set it when the used constructor is a `File`. Let me know if I should do that.